src/pages/Beta/Pool/AddLiquidityDrawer/index.tsx

```diff
@@ -27,7 +27,13 @@ const AddLiquidityDrawer: React.FC<Props> = ({ isOpen, onClose, onComplete, clic
   return (
     <Drawer title={t('pool.addLiquidity')} isOpen={isOpen} onClose={onClose} backgroundColor={backgroundColor}>
       {isOpen && (
-        <AddLiquidity currencyA={currencyA} currencyB={currencyB} onComplete={onComplete ?? onClose} type="card" />
+        <AddLiquidity
+          currencyA={currencyA}
+          currencyB={currencyB}
+          onComplete={onClose}
+          onAddToFarm={onComplete}
```

Comment:
line 33 and 34 is confusing because in onComplete we are passing onClose and in onAddToFarm we are passing onComplete. 

We added onComplete prop in this PR https://github.com/pangolindex/interface/pull/546 to handle redirecting to farm drawer. now lets rename onComplete with `onAddToFarm` in `AddLiquidityDrawer`

so it should be
`onAddToFarm={onAddToFarm}`

and 

`onComplete={onClose}` 

so we dont need `onComplete` prop in `AddLiquidityDrawer`